src/common/InfoLabel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Is it possible to have generalized components like Bootstrap has for such cases? For example a <WarningNotice /> component that can be supplied a text prop to show Offline notice, or any other warning messages. A <InformationNotice /> component to show general information like "Unread messages". Bootstrap has generalized warning, danger, info e.t.c